### PR TITLE
tiup: add additional_args

### PR DESCRIPTION
**Pull request by WalterWj**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/tiup/pull/2340
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -554,6 +554,7 @@ tispark_workers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `additional_args`：配置额外的 Prometheus Args 参数。该字段为一个数组，数组元素为 Prometheus 运行参数。例如：开启 Prometheus 热加载配置，配置 `--web.enable-lifecycle` 即可
```
Comment (shawn0915) [inline tiup/tiup-cluster-topology-reference.md:557]: 这个参数虽然merge 到 tiup master 代码了 (https://github.com/pingcap/tiup/pull/2340) ，但鉴于 tiup 未发版，目前终端用户无法直接用到这个参数，建议待包含相关代码的tiup发版后再增加这段文档，最好再标注下从tiup v？开始可以使用

Comment (WalterWj) [reply tiup/tiup-cluster-topology-reference.md:557]: 这个 PR 可以晚点合并。后续的 tiup 发版，我补充下 tiup 版本信息吧。

Comment (WalterWj) [reply tiup/tiup-cluster-topology-reference.md:557]: 这个 PR 应该会合并到  tiup 1.15.0 这个版本

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -587,6 +587,7 @@ tispark_workers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `additional_args`：配置额外的 Prometheus Args 参数。该字段为一个数组，数组元素为 Prometheus 运行参数。例如：开启 Prometheus 热加载配置，配置 `--web.enable-lifecycle` 即可(该功能需要 tiup 升级到 1.15.0 版本及以上)
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:590]:
```suggestion
- `additional_args`：从 TiUP v1.15.0 开始引入，用于配置 Prometheus 额外运行参数。该字段是一个数组，数组元素为 Prometheus 运行参数。例如，要开启 Prometheus 热加载功能，可以将该字段配置为 `--web.enable-lifecycle`。
```

**Comment by Oreoxmt** [comment]
I will merge this PR once TiUP v1.15.0 has been released.

**Review by Oreoxmt — APPROVED** [review]

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.3 label because the v7.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.3 and will no longer receive new updates

**Comment by WalterWj** [comment]
@shawn0915 @srstack Tiup 1.15.0 has been released and this pr can be merged.

**Comment by WalterWj** [comment]
When can this be merged? Or do you need me to do something here? @Oreoxmt

**Pull request merged**